AppCenterReactNativeShared/android/src/main/java/com/microsoft/appcenter/reactnative/shared/AppCenterReactNativeShared.java

```diff
@@ -58,6 +59,17 @@ public static synchronized void configureAppCenter(Application application) {
             AppCenterLog.debug(LOG_TAG, "Configure with secret.");
             AppCenter.configure(application, sAppSecret);
         }
+
+        /*
+         * In case when sStartAutomatically flag is set, we are sure that Auth service will be started
```

Comment:
I suggest the following comment.

> When sStartAutomatically flag is set to true, every service (analytics/auth/crashes/etc.) will be started by separate `AppCenter.start` call. If Auth module is used, call `doNotResetAuthAfterStart` to avoid resetting the auth token.